src/sentry/models/release.py

```diff
@@ -568,10 +568,12 @@ def set_commits(self, commit_list):
                 group.update(status=GroupStatus.RESOLVED)
                 metrics.incr('group.resolved', instance='in_commit', skip_internal=True)
 
-            resolved_with_commit.send_robust(
+            issue_resolved.send_robust(
                 organization_id=self.organization_id,
                 user=actor,
                 group=group,
+                project=group.project,
+                resolution_type='in_commit',
```

Comment:
why not just send `with_commit` here?

Reply:
This should be consistent with what we send in other places- we send `in_commit` in `group_index`

Reply:
in_commit is different from with_commit (see my previous comment)